### DISCUSSION TITLE
Is Suspense working with react-query?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [preactjs/preact#3375] [Q&A] [answered]
I try to use Suspense with react-query, it renders my UI but I see no fallback, is this supported?

**Comment by developit** [comment, marked as answer]
Do you happen to have an example? 

Preact might not show loading UI as early as React, I believe the default suspense implementation includes a delay period to avoid thrashing spinners during quick loading sequences.

**Reply by ivanjeremic** [reply]
Yes [https://codesandbox.io/s/delicate-bash-kp2cc](https://codesandbox.io/s/delicate-bash-kp2cc) I tried also on slow 3G

**Reply by JoviDeCroock** [reply]
Preact does not export Suspense only preact/compat does

**Reply by JoviDeCroock** [reply]
Also you weren't turning on Suspense in the options... https://codesandbox.io/s/loving-khayyam-p7g2i?file=/src/index.js

[Enabling Suspense react-query](https://react-query.tanstack.com/guides/suspense)
